PETWorks/ldiversity.py

```diff
@@ -0,0 +1,47 @@
+from typing import Dict
+
+import pandas as pd
+
+from PETWorks.attributetypes import QUASI_IDENTIFIER, SENSITIVE_ATTRIBUTE
+
+
+def measureLDiversity(
+    anonymizedData: pd.DataFrame,
+    attributeTypes: Dict[str, str],
+) -> list[int]:
+    qis = []
+    sensitiveAttributes = []
+    lValues = []
+
+    for attribute, value in attributeTypes.items():
+        if value == QUASI_IDENTIFIER:
+            qis.append(attribute)
+        if value == SENSITIVE_ATTRIBUTE:
+            sensitiveAttributes.append(attribute)
+
+    for index in range(len(sensitiveAttributes)):
+        columns = (
+            qis
+            + sensitiveAttributes[:index]
+            + sensitiveAttributes[index + 1:]
+        )
+        groups = anonymizedData.groupby(columns)
+
+        sensitiveAttribute = sensitiveAttributes[index]
+        lValues += [group[sensitiveAttribute].nunique() for _, group in groups]
+
+    return lValues
+
+
+def validateLDiversity(lValues: list[int], l: int) -> bool:
```

Comment:
According to [the pep8 style guide](https://peps.python.org/pep-0008/#names-to-avoid), one should avoid using `l` as a variable name because it looks like 1 or I. Could you rename it?